assets/app/view/game_page.rb

```diff
@@ -51,8 +51,10 @@ def load_game
       return if game_id == @game&.id &&
         ((!cursor && @game.actions.size == @num_actions) || (cursor == @game.actions.size))
 
+      # Hotseat doesn't have player ids, use names instead.
+      players = @game_data['players'].map { |p| [p['id'] || p['name'], p['name']] }.to_h
```

Comment:
you can just make the initializer take in an either an array of names or the hash. if an array of names, then you can do this logic

Reply:
done